apis/dba/v1alpha1/elasticsearch_modreq_types.go

```diff
@@ -25,32 +28,59 @@ const (
 	ResourcePluralElasticsearchModificationRequest   = "elasticsearchmodificationrequests"
 )
 
-// ElasticsearchModificationRequest defines a Elasticsearch database version.
+// ElasticsearchModificationRequest defines a Elasticsearch Modification Request object.
 
 // +genclient
-// +genclient:nonNamespaced
-// +genclient:skipVerbs=updateStatus
 // +k8s:openapi-gen=true
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:path=elasticsearchmodificationrequests,singular=elasticsearchmodificationrequest,shortName=esmodreq,categories={datastore,kubedb,appscode}
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.phase"
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
 type ElasticsearchModificationRequest struct {
 	metav1.TypeMeta   `json:",inline,omitempty"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 	Spec              ElasticsearchModificationRequestSpec   `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
 	Status            ElasticsearchModificationRequestStatus `json:"status,omitempty" protobuf:"bytes,3,opt,name=status"`
 }
 
-// ElasticsearchModificationRequestSpec is the spec for elasticsearch version
+// ElasticsearchModificationRequestSpec is the spec for ElasticsearchModificationRequest object
 type ElasticsearchModificationRequestSpec struct {
+	// Elasticsearch object reference
```

Comment:
I think it needs a `ModificationType` field

Upgrade
ScaleUp
ScaleDown

etc?